src/ansi-c/scanner.l

```diff
@@ -1389,6 +1389,14 @@ __decltype          { if(PARSER.cpp98 &&
                 return make_identifier();
             }
 
+  /* This is a clang extension */
```

Comment:
Could a pointer to documentation be provided? Some day we may want to implement this.